src/controllers/router-store/index.tsx

```diff
@@ -63,65 +61,32 @@ const actions: AllRouterActions = {
     props =>
     ({ setState, dispatch }) => {
       const {
+        basePath = '',
+        history,
+        initialRoute,
+        onPrefetch,
         resourceContext,
         resourceData,
-        basePath = '',
         routes,
-        initialRoute,
-        ...initialProps
       } = props;
-      const { history, isStatic } = initialProps;
       const routerContext = findRouterContext(
         initialRoute ? [initialRoute] : routes,
         { location: history.location, basePath }
       );
 
       setState({
-        ...initialProps,
         ...routerContext,
         basePath,
+        history,
+        onPrefetch,
         routes,
         location: history.location,
         action: history.action,
       });
-      getResourceStore().actions.hydrate({ resourceContext, resourceData });
 
-      if (!isStatic) {
-        dispatch(actions.listen());
-      }
-    },
-
-  /**
-   * Duplicate method that uses isServerEnvironment instead of removed isStatic prop
-   * internally. We can remove this when UniversalRouter replaces Router completely.
-   */
-  bootstrapStoreUniversal:
-    props =>
-    ({ setState, dispatch }) => {
-      const {
-        resourceContext,
-        resourceData,
-        basePath = '',
-        ...initialProps
-      } = props;
-      const { history, routes } = initialProps;
-      const routerContext = findRouterContext(routes, {
-        location: history.location,
-        basePath,
-      });
-
-      setState({
-        ...initialProps,
-        ...routerContext,
-        basePath,
-        location: history.location,
-        action: history.action,
-      });
       getResourceStore().actions.hydrate({ resourceContext, resourceData });
 
-      if (!isServerEnvironment()) {
-        dispatch(actions.listen());
-      }
+      dispatch(actions.listen());
```

Comment:
No need to prevent this from being called on server?

Reply:
Ah I think this was ported incorrectly from your original changes 😂

Reply:
Should be fixed now